bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -51,14 +51,23 @@
     <section id="vr" class="odd">
       <div class="content">
         <div class="inner-container">
+        {% if l10n_has_tag('mozorg_tech_webvr') %}
+          <h2>{{ _('Bringing Virtual Reality to the web platform') }}</h2>
+          <p>{{ _('Using WebVR, developers, designers and artists can share VR experiences on the web.') }}</p>
+          <a rel="external" href="https://vr.mozilla.org/" class="button">{{ _('Learn more') }}</a>
+        {% else %}
           <h2>{{ _('Making Virtual Reality a reality for all') }}</h2>
           <p>{{ _('Using A-Frame, developers, designers and artists are able to easily create accessible VR experiences.') }}</p>
           <a rel="external" href="https://aframe.io/" class="button">{{ _('Explore A-Frame') }}</a>
+        {% endif %}
         </div>
       </div>
       <footer>
         <ul>
-        {% if l10n_has_tag('emerging_tech_links') %}
+        {% if l10n_has_tag('mozorg_tech_webvr') %}
+          <li><a rel="external" href="https://research.mozilla.org/mixed-reality/">{{ _('Learn about WebVR') }}</a></li>
+          <li><a rel="external" href="https://aframe.io/">{{ _('Explore A-Frame') }}</a></li>
```

Comment:
It looks like we already have the string `Explore A-Frame` translated above. Could we avoid adding another conditional here, seeing as `Learn about WebVR` is also in the l10n block below?